dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/DynamicSimulationProvider.java

```diff
@@ -18,6 +18,6 @@
  */
 public interface DynamicSimulationProvider extends Versionable, PlatformConfigNamedProvider {
```

Comment:
Just a question about the _semantics_ of the `Versionable` interface.

I assume the name and version strings returned by a `DynamicSimulationProvider` should be interpreted as the name and version of the external simulator that this provider is integrating in PowSyBl ?

For example for Dynawo, `getName()` would return `"Dynawo"` and `getVersion()` `"1.1.0"`. 

Or should it be related to a specific version of the provider, internal to powsybl, and used consistently for later potential updates of the same "kind" of provider ?

As an example: we start with a Dynawo provider with version `"1"` that connects only with Dynawo 1.1.0. Later we create an updated Dynawo provider, version `"2"`, that is able to connect with multiple versions of Dynawo, starting from 1.1.x and up to Dynawo 2.0.0.

How the version attribute is planned to be used in PowSyBl ?

Reply:
No idea how we'll use it in powsybl. From my point of view, the provider should use this to ensure that it is compatible with the binary it will run. It's how we use it in RTE for Hades2.

But I don't know if we already discuss (and agree) on how it will be used. `Versionable` is a really old interface... coming from iTesla project.